core/src/test/java/org/apache/calcite/tools/FrameworksTest.java

```diff
@@ -292,6 +293,37 @@ private void checkTypeSystem(final int expected, FrameworkConfig config) {
         });
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-3228">[CALCITE-3228]
+   * Error while applying rule ProjectScanRule:interpreter</a> */
```

Comment:
can the comment be more explicative "Error while applying rule ProjectScanRule:interpreter" is not telling that the problem is with an aggregate query without "group by"

Reply:
Ok. Will do.

Reply:
@chunweilei @eolivelli  I tested the unit test against this branch and it works for **"SELECT MIN(n1) as mi, MAX(n1) as ma FROM tblspace1.tsql WHERE k1='no_results' GROUP BY k1"** query. But the **UPDATE tblspace1.table1 set n1=1000 WHERE k1 in (SELECT fk FROM tblspace1.table2 WHERE k2=10)** query still fails

Reply:
See https://github.com/apache/calcite/pull/1358 for the UPDATE query.